GravityPDF/gravity-pdf
# Add GitHub Actions

## Description

TravisCI will begin charging at the end of the year, so we're migrating to GitHub Actions.

## Testing instructions
Go to Actions tab above and see that everything checks out. 

Should also see comment below once everything completes and is working (or not working). 

## Checklist:
